### DISCUSSION TITLE
06/17 안건

### DISCUSSION THREAD

**Discussion by YuHyun-P** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#161] [Announcements]
- [x] #160 
- [x] #162

**Comment by hyunjin0910** [comment]
- [ ] [변경값 있는 경우 context 정보 관리](https://github.com/prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung/pull/151) #151

**Reply by Hyevvy** [reply]
현진님 궁금한게 로그인 정보에 대한 Context는 별도로 만드시는건가용??
